src/client/components/pages/entity-revisions.js

```diff
@@ -83,6 +107,12 @@ class EntityRevisions extends React.Component {
 		);
 	}
 
+/**
+* Renders the EntityRevisions page, which is a list of all the revisions made
+* to an entity, along with information about the author and the
+* first revision note for each revision.
+* @returns {ReactElement} a HTML document which is a part of the Revision page
```

Comment:
"a HTML document which displays the Revision page" - this renders the whole thing, not just a part